test/fixtures/azure-fixture/src/main/java/fixture/azure/AzureHttpFixture.java

```diff
@@ -49,16 +49,22 @@ public enum Protocol {
         HTTPS
     }
 
-    public static Predicate<String> startsWithPredicate(String expectedPrefix) {
+    /**
+     * @param  account The name of the Azure Blob Storage account against which the request should be authorized..
+     * @return a predicate that matches the {@code Authorization} HTTP header that the Azure SDK sends when using shared key auth (i.e.
+     *         using a key or SAS token).
+     * @see <a href="https://learn.microsoft.com/en-us/rest/api/storageservices/authorize-with-shared-key">Azure docs on shared key auth</a>
```

Comment:
Hope this helps clarify - the header is coming from the Azure SDK, and I'm working towards a change that should cause it to emit a different kind of auth header, so I want to add these assertions now in order to better support the upcoming change.

Reply:
Very helpful 👍